docs/framework/01intro/introduction.md

```diff
@@ -1,33 +1,26 @@
 # Introduction
+With several layer-one protocols gaining meaningful traction over the last couple of years and layer-two protocols taking [center stage in the scalability roadmap for Ethereum](https://vitalik.ca/general/2021/12/06/endgame.html), it is increasingly becoming clear that the future will be multichain. This thesis is further reinforced by emerging patterns such as modular blockchains, application-specific chains, and fractal scaling. A multichain world is one in which disparate chains co-exist with thriving ecosystems. 
```

Comment:
>A multichain world is one in which disparate chains co-exist with thriving ecosystems.

with thriving... 
or 
as thriving...

That is, are the chains the ecosystems, or are the ecosystems something else?

Reply:
The ecosystems in this context are the chains and applications built on top of them. I think your suggested wording sounds more fitting. I've made the change